src/System.Private.CoreLib/shared/System/AppDomain.cs

```diff
@@ -195,15 +195,13 @@ public static bool MonitoringIsEnabled
             }
```

Comment:
I am wondering whether we should also make `MonitoringIsEnabled` return `true`, and delete `throw new PlatformNotSupportedException(SR.PlatformNotSupported_AppDomain_ResMon);` from the setter.

Then the only wrinkle that remains is `MonitoringTotalProcessorTime` that is "just work" to implement: using `GetProcessTimes` and `getrusage`. This can be separate PR.

Reply:
Good idea.  Done.